src/global.interface.ts

```diff
@@ -37,6 +37,8 @@ export type GlobalKeyboardEventKey =
   | 'ArrowLeft'
   | 'ArrowUp'
   | 'ArrowDown'
+  | 'Meta'
+  | 'Control'
```

Comment:
Can you write a simple comment like this:
```suggestion
  | 'Meta' // ⌘ in Apple Ecosystem
  | 'Control'
```